bool run_flag
float32 ent
float32 pX
float32 pY
float32 pZ
float32 rX
float32 rY
float32 rZ
float32 el_dir
float32 motion_type
float32 next_tool
float32 mag_range
float32 mag_x
float32 mag_y
float32 mag_z
float32 gripper

